pkg/crypto/tinkcrypto/primitive/jwe/subtle/ecdhes/ecdhes_kw_aead_jwe_encrypt.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package ecdhes
+
+import (
+	"crypto/ecdsa"
+
+	"github.com/google/tink/go/subtle/hybrid"
+	"github.com/square/go-jose/v3"
+)
+
+// KWEncrypt is an instance of ECDH-ES With ConcatKDF KW (Key Wrapping) JWE encryption.
+type KWEncrypt struct {
+	recipientPubKey *hybrid.ECPublicKey
+	pointFormat     string
+	// encHelper is only added to bypass Tink for crypto
+	encHelper EncryptHelper
+	// this implementation uses go-jose to encrypt and build a JWE instance
+	encrypter jose.Encrypter
+}
+
+// NewECDHESEncrypt returns ECDH-ES encryption construct with Concat KDF/ECDH-ES key wrapping
+// and AEAD payload encryption in a JWE format.
+func NewECDHESEncrypt(recipientKey *hybrid.ECPublicKey, ptFormat string, encHelper EncryptHelper) (*KWEncrypt, error) {
+	c, err := hybrid.GetCurve(recipientKey.Curve.Params().Name)
+	if err != nil {
+		return nil, err
+	}
+
+	recipientPubKey := &hybrid.ECPublicKey{
+		Curve: c,
+		Point: recipientKey.Point,
+	}
+
+	pubKey := &ecdsa.PublicKey{
+		Curve: recipientPubKey.Curve,
+		X:     recipientPubKey.Point.X,
+		Y:     recipientPubKey.Point.Y,
+	}
+
+	// for now, assuming recipient kw alg is jose.ECDH_ES_A256KW using pubKey provided by Tink
+	rcpt := &jose.Recipient{Algorithm: jose.ECDH_ES_A256KW, Key: pubKey}
+
+	encOpts := &jose.EncrypterOptions{
+		Compression: jose.NONE,
+	}
+
+	// for now, assuming didcomm type, TODO: jwe type will need to be parametrized
```

Comment:
Can we fix this asap? This package should not be hardcoded to didcomm.

Reply:
sure.. I wanted to get something out as soon as possible. 

the same for encHelper and encrypter, they should be merged, probably encrypter should be a member of encHelper, I'll update this as well.

Reply:
outdated code and therefore removed